glide.yaml

```diff
@@ -1,3 +1,22 @@
+# yours github.com/containers/storage
+# mine github.com/getsentry/raven-go
+# mine github.com/gorilla/securecookie
+# mine github.com/gorilla/sessions
+# mine github.com/openshift/api
+# mine github.com/openshift/client-go
+# mine github.com/pkg/profile
+# mine github.com/RangelReale/osin
+# ours github.com/apcera/gssapi
+# ours k8s.io/api
+# ours k8s.io/apiextensions-apiserver
+# ours k8s.io/apimachinery
+# ours k8s.io/apiserver
+# ours k8s.io/client-go
+# ours k8s.io/code-generator
+# ours k8s.io/kube-aggregator
+# ours k8s.io/kubernetes
+# ours k8s.io/metrics
```

Comment:
@deads2k is this at all appromixating what you wanted?

i didn't examine the k8s ones, i just assumed they would all be "ours"
also assumed the openshift ones would be "mine"

Reply:
> @deads2k is this at all appromixating what you wanted?
> 
> i didn't examine the k8s ones, i just assumed they would all be "ours"
> also assumed the openshift ones would be "mine"

Yes, exactly.  Thanks.